src/ScClient.ts

```diff
@@ -1,48 +1,62 @@
 import { invalidValue } from "./errors";
 import { ScAddr } from "./ScAddr";
 import { ScConstruction } from "./ScConstruction";
-import { ScConstructionCommand } from "./ScConstructionCommand";
-import { ScEvent, ScEventCallbackFunc } from "./scEvent";
+import { ScEvent } from "./scEvent";
 import { ScEventParams } from "./ScEventParams";
-import { ScLinkContent, ScLinkContentType } from "./ScLinkContent";
+import {
+  ScLinkContent,
+  ScLinkContentType,
+  TContentString,
+} from "./ScLinkContent";
 import { ScTemplate, ScTemplateTriple, ScTemplateValue } from "./ScTemplate";
-import { ScTemplateSearchResult, ScTemplateResult } from "./ScTemplateResult";
+import { ScTemplateResult } from "./ScTemplateResult";
 import { ScType } from "./scType";
```

Comment:
```suggestion
import { ScType } from "./ScType";
```
Please update for all references